data/json/recipes/weapon/cutting.json

```diff
@@ -29,8 +30,8 @@
     "time": 55000,
     "book_learn": [ [ "textbook_armschina", 5 ] ],
     "proficiencies": [
-      { "proficiency": "prof_blacksmithing", "required": false, "time_multiplier": 3.0 },
-      { "proficiency": "prof_bladesmith", "required": false, "time_multiplier": 2.5 }
+      { "proficiency": "prof_metalworking", "required": false, "time_multiplier": 1.5, "fail_multiplier": 5 },
+      { "proficiency": "prof_redsmithing", "required": false, "time_multiplier": 3 }
```

Comment:
Was this branched off another PR? This and other changes seems out of the view of a resort.

Reply:
I started working on cutting.json for proficiencies before realizing there were only about four recipes in there. I could take it out, it's not a big deal.